src/app/shared/state/vstream/vstream.feature.ts

```diff
@@ -104,22 +113,27 @@ const initialCustomMessage: VStreamCustomMessageState = {
   enabledChat: false,
 };
 
-const initialChatCommand: Omit<ChatCommand, 'id'> = {
-  command: '!command',
-  permissions: {
-    allUsers: true,
-    payingMembers: false,
-    mods: false,
-  },
-  autoRedeemInterval: 0,
-  autoRedeem: false,
-  cooldown: 0,
-  enabled: true,
-  response: 'Change me!',
-};
+/**
+ * Return a default state command for the passed in type.
+ * @param type
+ */
+function changeCommandType(type: CommandTypes) {
```

Comment:
I _always_ want these to be object maps instead but every benchmark I've seen says a switch or if-else is faster 😢